tests/reg_issue1352/issue_1352.fz.expected_err

```diff
@@ -19,4 +19,13 @@ with argument count of the lambda expression equal to the number of type paramet
 assigned field must be given explicitly.
 To solve this, declare an explicit type for the target field, e.g., 'f (i32, i32) -> bool := x, y -> x > y'.
 
-3 errors.
+
+--CURDIR--/issue_1352.fz:27:10: error 4: No type information can be inferred from a lambda expression
+  d => b ->
+---------^
```

Comment:
Would be great if this would appear instead of `error 2` reported above. But this could be fixed separatedly in a new PR.